packages/components/molecules/f-media-element/src/components/MediaElement.vue

```diff
@@ -1,18 +1,120 @@
 <template>
     <div
-        :class="$style['c-mediaElement']"
-        data-test-id="mediaElement-component">
-        {{ componentName }}
+        data-test-id="mediaElement-component"
+        :class="[$style['c-mediaElement'], { [$style['c-mediaElement--stack']]: stacked }, { [$style['c-mediaElement--reverse']]: reverse }]">
```

Comment:
Would be good to split this out to a multi-line construct, for readability. I believe you should also merge the objects here into one:

```suggestion
        :class="[$style['c-mediaElement'], {
            [$style['c-mediaElement--stack']]: stacked,
            [$style['c-mediaElement--reverse']]: reverse
        }]">
```

Reply:
agreed, will follow what ash has suggested for this

Reply:
fixed